test/input_files/failed/failed_fileformat_000.vcf

```diff
@@ -1,4 +1,4 @@
-##fileformat=
+##qformat=
```

Comment:
An empty fileformat is already an error. If you want to add a check for using an incorrect string please create a new test file.